coordinator/internal/orm/chunk.go

```diff
@@ -161,9 +161,6 @@ func (o *Chunk) getLatestChunk(ctx context.Context) (*Chunk, error) {
 
 	var latestChunk Chunk
 	if err := db.First(&latestChunk).Error; err != nil {
-		if errors.Is(err, gorm.ErrRecordNotFound) {
```

Comment:
Decided to undo this change?

Reply:
Yes. This would not happen in a non-genesis path, thus `not found` needs to be considered as an error case, instead of handling a `nil` chunk outside.